language.md

```diff
@@ -1,5 +1,7 @@
 ﻿If you speak English, [go here](english/marshmallow.md).
 
+Má labhraíonn tú Gaeilge, [téigh anseo](irish/start).

```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.